spec/defines/requirements_spec.rb

```diff
@@ -42,8 +45,16 @@
       end
 
       context 'default' do
+        let :params do
+          {
+            'cwd': '/foo'
+          }
+        end
         it { is_expected.to contain_file('/requirements.txt').with_owner('root').with_group('root') }
       end
+      describe 'without cwd' do
```

Comment:
I'd visually separate these
```suggestion

      describe 'without cwd' do
```